scipy/signal/filter_design.py

```diff
@@ -715,7 +715,7 @@ def sosfreqz(sos, worN=512, whole=False, fs=2*pi):
     r"""
     Compute the frequency response of a digital filter in SOS format.
 
-    Given `sos`, an array with shape (n, 6) of second order sections of
+    Given ``sos``, an array with shape (n, 6) of second order sections of
```

Comment:
Reverting to check if it still fails without these:

```suggestion
    Given `sos`, an array with shape (n, 6) of second order sections of
```